MarioGame/Source/Scenes/LevelScene.cs

```diff
@@ -181,6 +182,11 @@ public void Update(GameTime gameTime, SceneManager sceneManager)
             {
                 var playerPosition = playerEntity.GetComponent<ColliderComponent>().Position;
                 LoadEntitiesNearPlayer(playerPosition, LoadRadius);
+
+                if (IsPlayerAtSecretLocation(3620, 3776))
```

Comment:
Fix the position limits of the ducts, at x=3776 Mario is out of the duct and he tele transports anyways
![image](https://github.com/Nintentando-Team-Software-Developmet/MarioBrosGame/assets/135129192/29c82eb8-1df5-4ba2-bd5c-b90c7b47cce9)

Reply:
That part is from another US in this case that is only implemented to test the scene loading in general.